app/views/journal_closing_reminders/index.html.haml

```diff
@@ -0,0 +1,26 @@
+= content_for :h1 do
+  = t("pages.global_settings")
+
+= content_for :sidebar do
+  = render "admin/shared/sidenav_global", sidenav_tab: "journal_closing_reminders"
+
+%h2= JournalClosingReminder.model_name.human(count: 2)
+
+= link_to t(".new"), new_journal_closing_reminder_path, class: "btn-add"
+
+%table.table
+  %thead
+    %th
+    %th= t(".start_date")
+    %th= t(".end_date")
+    %th= t(".fiscal_year")
+    %th= t(".message")
+  %tbody
+    - @journal_closing_reminders.each do |journal_closing_reminder|
+      %tr{class: journal_closing_reminder.past? ? "row-warning" : "" }
+        %td= link_to t(".edit"), edit_journal_closing_reminder_path(journal_closing_reminder)
```

Comment:
might be where you'd have scope="row" but it would also need unique value, like "edit reminder #1" or something

Reply:
I thought scope=row is for a `th` element, is that right?

Reply:
yes, that would be correct, and there's typically a header field on a row, which i was determining to be the edit field